Expensify/App
# For avatar picker restrict only with image file types

@tgolen 

### Details
Restricted Avatar picker to pick only image file types.

### Fixed Issues

$ #4645 

### Tests && QA Steps
1. Go to Settings
2. Click Profile
3. Click the pen icon to update or add a photo
4. Click upload photo
5. Refer **Fix** section below, it should work as shown there.
6. Create a new workspace if you don't have one.
7. Go to Setting 
8. Click on your workspace.
9. Tap on the Workspace icon  
10. Click the pen icon to update or add a photo
11. Click upload photo
12. Refer **Fix** section below, it should work as shown there.

#### Fix:
##### In Desktop or web:
The file picker opens and will allow picking only image file types now.

##### In Mobile, we should show only two options now
- Take photos,
- Gallery,
-  ~Choose Document~ Should be removed 

#### Mobile web
Options may vary based on the OS Type - but should be able to pick images only
Select Gallery should show only the photos

### Tested On

### Screenshots


#### Web
<img width="1433" alt="Screenshot 2021-09-05 at 4 35 34 PM" src="https://user-images.githubusercontent.com/85645967/132126027-4bfd1d7d-2bc2-40a4-8047-2edfae301ac8.png">
<img width="1431" alt="Screenshot 2021-09-05 at 4 34 06 PM" src="https://user-images.githubusercontent.com/85645967/132126040-a51c4a2a-fb78-4ad9-bddc-760c2e632600.png">

#### Mobile Web
![Simulator Screen Shot - iPhone 11 - 2021-09-05 at 18 11 02](https://user-images.githubusercontent.com/85645967/132127158-fd1fd1a7-16ef-47b4-bc97-ef486627e700.png)
![Simulator Screen Shot - iPhone 11 - 2021-09-05 at 18 11 06](https://user-images.githubusercontent.com/85645967/132127160-085faa5f-6431-4f66-9b56-ad17feca28cc.png)

#### Desktop
<img width="1191" alt="Screenshot 2021-09-05 at 4 21 48 PM" src="https://user-images.githubusercontent.com/85645967/132126053-e7537742-27fd-455a-8602-91b45b8e16c1.png">
<img width="1184" alt="Screenshot 2021-09-05 at 4 21 03 PM" src="https://user-images.githubusercontent.com/85645967/132126057-8e9b39a0-e90b-436e-b3cd-ad21c1f19f0d.png">

#### iOS
![Simulator Screen Shot - iPhone 12 - 2021-09-05 at 16 31 18](https://user-images.githubusercontent.com/85645967/132126190-f71c4126-6787-4df3-86b4-b3a4b00cd88f.png)
![Simulator Screen Shot - iPhone 12 - 2021-09-05 at 16 30 02](https://user-images.githubusercontent.com/85645967/132126196-d5a1283c-9b6c-4030-ba64-cc7e8b124595.png)

#### Android
![Screenshot_1630839652](https://user-images.githubusercontent.com/85645967/132126202-7331f173-ef04-4fa5-93bd-69972bb371dd.png)
![Screenshot_1630839665](https://user-images.githubusercontent.com/85645967/132126206-309d8da9-25ae-4098-ace5-cae12620f392.png)
